sass/responsive/_global.scss

```diff
@@ -0,0 +1,66 @@
+.app__body {
+    color: var(--center-channel-color);
+    scrollbar-arrow-color: var(--center-channel-color);
+
+    .a11y--focused {
+        box-shadow: inset 0 0 1px 3px rgba(var(--link-color-rgb), 0.5), inset 0 0 0 1px var(--link-color);
+    }
+
+    hr {
+        border-color: rgba(var(--center-channel-color-rgb), 0.2);
+    }
+
+    a,
+    .color-link {
+        color: var(--link-color);
+
+        &:focus,
+        &:hover, {
+            color: var(--link-color);
+        }
+    }
+
+    .input-group-addon {
+        border-color: rgba(var(--center-channel-color-rgb), 0.1);
+        background: rgba(var(--center-channel-color-rgb), 0.05);
+    }
+
+    .svg-text-color,
+    .icon--body {
+        fill: var(--center-channel-color);
+    }
+
+    #post-create,
```

Comment:
I am generally not a fan of having random selectors bunched up in a global style as it makes it very difficult to trace where the change is coming from when debugging. Can we keep the global style relatively global? And split up component specific styles in the module of that component.

Reply:
yes, will try to do that